src/main/java/com/sessions/session9/homeworkOOP2/User.java

```diff
@@ -0,0 +1,158 @@
+package com.sessions.session9.homeworkOOP2;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.Random;
+
+public class User implements Comparable<User>
```

Comment:
Foarte tare ca ai folosit Comparable. Daca vrei sa treci deja la next level in compararea de obiecte iti recomand sa te uiti si peste Comparator daca nu ai facut-o deja:
https://www.baeldung.com/java-comparator-comparable